# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/ipc/WindowGlobalChild.h
# Commit: 459739221a88
# Full Hash: 459739221a884b2b1c2adab42fd19fafe279bd20
# Author: Sandor Molnar <smolnar@mozilla.com>
# Date: 2025-06-05 04:07:37
# Regressor Bug: 1969017
# File Overlap Count: 2
# Description:
#   Revert "Bug 1969015, 1969017: apply code formatting via Lando" for causing build bustages
#   
#   This reverts commit ab681e36aeafc21ba62bb4680bdaa6206f02546d.
#   
#   Revert "Bug 1969017 - Refactor FedCM to use a managed actor rather than tagging onto WindowGlobalParent - r=anti-tracking-reviewers,emz"
# ==============================================================================

diff -r fc9986230203 -r 459739221a88 dom/ipc/WindowGlobalChild.h
--- a/dom/ipc/WindowGlobalChild.h	Thu Jun 05 00:15:19 2025 +0300
+++ b/dom/ipc/WindowGlobalChild.h	Thu Jun 05 00:19:41 2025 +0300
@@ -201,6 +201,9 @@
   mozilla::ipc::IPCResult RecvNotifyPermissionChange(const nsCString& aType,
                                                      uint32_t aPermission);
 
+  mozilla::ipc::IPCResult RecvNavigateForIdentityCredentialDiscovery(
+      const nsCString& aURI, const IdentityLoginTargetType& aType);
+
   virtual void ActorDestroy(ActorDestroyReason aWhy) override;
 
  private: